actions/AuthAction.js

```diff
@@ -66,7 +66,7 @@ async function checkForProductContext(endpoint, clientId, org, token, productCon
 
 function asAuthAction(action) {
   return async (params) => {
-    const logger = Core.Logger('main', { level: params.LOG_LEVEL || 'info' });
+    const logger = Core.Logger('AuthAction', { level: params.LOG_LEVEL || 'info' });
```

Comment:
Instead of changing the log level through a parameter, which would require altering the frontend code, can we set it using an environment variable during deployment, as outlined in the AIO [documentation](https://github.com/adobe/aio-lib-core-logging?tab=readme-ov-file#configuration) (via AIO_LOG_LEVEL)? This should also be made a global object, like the modifications in FirefallClient. We should also remove the configuration part entirely. This way, the AIO logger will use the environment variable for the log level. We also need to add this variable to the GitHub Actions setup.

Reply:
@vtsaplin I've made modifications to incorporate your feedback.